client/src/ui/elements/ResourceCost.tsx

```
@@ -60,7 +61,7 @@ export const ResourceCost = ({
             maximumFractionDigits: 1,
           }).format(props.amount || 0)}{" "}
           <span className={clsx(balanceColor, "font-normal")}>
-            {!isNaN(balance) && `(${formatNumber(balance, 0)})`}{" "}
+            {!isNaN(balance) && `(${currencyFormat(balance, 0)})`}{" "}
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)